src/praisonai-agents/praisonaiagents/scheduler/config_store.py

```
@@ -71,27 +83,166 @@ def list(self, agent_id: Optional[str] = None) -> List[ScheduleJob]:
+                try:
+                    if locked:
+                        try:
+                            import fcntl  # type: ignore[import-not-found]
+                            fcntl.flock(fh.fileno(), fcntl.LOCK_UN)
+                        except ImportError:
+                            try:
+                                import msvcrt  # type: ignore[import-not-found]
+                                fh.seek(0)
+                                msvcrt.locking(fh.fileno(), msvcrt.LK_UNLCK, 1)
+                            except Exception:  # pragma: no cover
+                                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)